Source/Core/Core/AchievementManager.cpp

```diff
@@ -95,6 +94,13 @@ void AchievementManager::LoadGameByFilenameAsync(const std::string& iso_path,
     callback(AchievementManager::ResponseType::MANAGER_NOT_INITIALIZED);
     return;
   }
+  if (m_disabled)
+  {
+    INFO_LOG_FMT(ACHIEVEMENTS, "Achievement Manager is disabled until core is rebooted.");
+    OSD::AddMessage("Achievements are disabled until core is rebooted.", OSD::Duration::VERY_LONG,
```

Comment:
```suggestion
    OSD::AddMessage("Achievements are disabled until you restart emulation.", OSD::Duration::VERY_LONG,
```

The concept of the "core" is not something the user normally is told about.